bin/unipoints

```diff
@@ -157,10 +157,10 @@ function main(array $args): int
 }
 
 try {
-    $rc = main($argv);
+    $exitStatus = main($argv);
 } catch (UserMessageException $x) {
     echo trim($x->getMessage()), "\n";
-    $rc = 1;
+    $rc = exitStatus;
```

Comment:
I guess it's a typo? It should be $exitStatus = 1

Reply:
You see? I don't write code daily.